tests/test_parse.py

```diff
@@ -106,6 +107,24 @@ def test_file_json_no_ext(tmpdir):
     assert Model.parse_file(str(p)) == Model(a=12, b=8)
 
 
+def test_file_json_loads(tmpdir):
```

Comment:
I'd prefer to use `tmp_path` since it matches the modern `Path` iterface, you can also omit the `str(p)` below.

Reply:
Yep. I hadn't spotted the `tmp_path` - it's much nicer.

Reply:
yup, I learn most new things from contributing to open source and people contributing to my projects, hence why I try to give small pointers like this.